parsec-gradle-plugin/build.gradle

```diff
@@ -72,8 +66,29 @@ task downloadParsecRdl(type: Exec) {
     }
 }
 
-jar.dependsOn downloadParsecRdl
-publishPlugins.dependsOn downloadParsecRdl
-publishPlugins.mustRunAfter check
-postProcess.dependsOn publishPlugins
-test.dependsOn downloadParsecRdl
+task downloadRdlBinary(type: Exec) {
+    def osName = System.getProperty("os.name").equals("Mac OS X") ? "darwin" : "linux";
```

Comment:
does "equals("Mac OS X")" here mean this check is limited to the specific version of Mac OS ?
would it be better to just check "SystemUtils.IS_OS_MAC"?

reference:
http://stackoverflow.com/questions/4689240/detecting-the-platform-window-or-linux-by-groovy-grails